tests/System/integration/administrator/components/com_config/Application.cy.js

```diff
@@ -24,7 +24,7 @@ describe('Test in backend that the application configuration', () => {
     cy.task('getMails').then((mails) => {
       cy.get('#system-message-container').should('contain.text', 'The email was sent to');
 
-      cy.wrap(mails).should('have.lengthOf', 1);
+      expect(mails.length, 'There is not just one mail, you may need to check the SMTP configuration').to.equal(1);
```

Comment:
assuming I have understood this correctly

```suggestion
      expect(mails.length, 'An email was not sent, you may need to check the SMTP configuration of the server.').to.equal(1);
```

Reply:
i can understand this is not clear to read, i fiddled a bit with the expression, because strictly speaking the test would also fail if, for example, 2 emails arrived, i didn't found a better expression

_An email was not sent_ may therefore not be correct. And _check the SMTP configuration of the server_ is not what is needed. It may be the Cypress SMTP configuration with `smtp_port` and `smtp_host` to check that this port is unused and open in the firewall, if there is one.

this will be documented in tests/System/README.md

and you have to consider the overall assertion error message:
`AssertionError: There is not just one mail, you may need to check the SMTP configuration: expected 0 to equal 1`

and I am open for better expression ...

Reply:
Hey @brianteeman i would like to solve this PR together with you as you are the right focus person 😃 So, we have three situations, 0, 1 or 2 mails are expected. Could you please make your choice:

A. error as it is actual implemented with
  - AssertionError: There is not just one mail, you may need to check the SMTP configuration: expected 0 to equal 1
  - AssertionError: There are not two mails, you may need to check the SMTP configuration: expected 0 to equal 2
  - AssertionError: There should be no mail: expected 42 to equal 0
 
B. without the individual error message
  - AssertionError: expected 0 to equal 1
  - AssertionError: expected 0 to equal 2
  - AssertionError: expected 42 to equal 0
 
C. your choice
  - (in case exactly one email expected)
  - (in case exactlytwo emails expected)
  - (in case no emails expetced)

Thank you in advance and greetings from Berlin area
Heiko

Reply:
ok, no answer after a few days, so I'll stick with A.  Error messages as they are currently implemented and resolve this conversation to be ready for merge